Ginger/Ginger/ALM/JIRA/JiraZephyrCyclesExplorerPage.xaml.cs

```diff
@@ -0,0 +1,259 @@
+                {
+                    mCurrentSelectedCycles.Add((JiraZephyrCycleTreeItem)mCurrentSelectedTreeItem);
+                }
+            }
+        }
+
+        private void GetTestSetDetails(QCTestSetTreeItem testSetItem)
+        {
+            mExecDetailNames = testSetItem.TestSetStatuses;
+            LoadExecutionDetails();
+            if (testSetItem.AlreadyImported)
+            {
+                lblTSAlreadyImported.Content = "Yes";
+                txtBoxTSMappedBF.Text = "'" + testSetItem.MappedBusinessFlowPath + "'";
+            }
+            else
+            {
+                lblTSAlreadyImported.Content = "No";
+                txtBoxTSMappedBF.Text = "None";
+            }
+        }
+
+        private void ShowCycleDetailsPanel(bool toShow)
+        {
+            if (toShow)
+            {
+                CycleDetailsPanel.Visibility = System.Windows.Visibility.Visible;
+                testSetDetailsColumn.Width = new GridLength(200);
+            }
+            else
+            {
+                CycleDetailsPanel.Visibility = System.Windows.Visibility.Collapsed;
+                testSetDetailsColumn.Width = new GridLength(0);
+            }
+        }
+
+        public object ShowAsWindow(eWindowShowStyle windowStyle = eWindowShowStyle.Dialog)
+        {
+            if (mParentSelectionMode)
+            {
+                Button importBtn = new Button();
+                importBtn.Content = "Select";
+                importBtn.Click += new RoutedEventHandler(SelectFolder);
+                GingerCore.General.LoadGenericWindow(ref _GenericWin, App.MainWindow, windowStyle, "Select Path For Export", this, new ObservableList<Button> { importBtn }, true, "Cancel", Cancel_Clicked);
+                return mCurrentSelectedObject;
+            }
+            else
+            {
+                Button importBtn = new Button();
+                importBtn.Content = "Import Selected";
+                importBtn.Click += new RoutedEventHandler(ImportSelected);
+                GingerCore.General.LoadGenericWindow(ref _GenericWin, App.MainWindow, windowStyle, "Browse JiraZephyr Cycles", this, new ObservableList<Button> { importBtn }, true, "Cancel", Cancel_Clicked);
+                return CurrentSelectedVersions;
+            }
+        }
+
+        private void SelectFolder(object sender, RoutedEventArgs e)
+        {
+            if (mCurrentSelectedTreeItem == null)
+            {
+                Reporter.ToUser(eUserMsgKey.StaticWarnMessage, "Please select a folder item");
+            }
+            else
+            {
+                _GenericWin.Close();
+            }
+        }
+
+        private void ImportSelected(object sender, RoutedEventArgs e)
+        {
+            if (mCurrentSelectedTreeItem != null)
+            {
+                if (ALMIntegration.Instance.ImportZephyrCycle(mImportDestinationPath, (IEnumerable<object>)CurrentSelectedCycles));
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)